### PR TITLE
Docs: Update colab guide

### PR DESCRIPTION
**Pull request by dvsrepo**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes #<issue_number>

**Type of change**
- [x] Documentation update

### CODE REVIEW

File: docs/_source/getting_started/installation/deployments/google-colab.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
 
-To use Argilla with Google Colab, the recommended method is to use Hugging Face Spaces, as [described in this guide](huggingface-spaces.md). This is the easiest way to set up Argilla for use with Google Colab. For other installations, we recommend you to read the [cloud providers guide](cloud_providers.md).
+You can run all Argilla tutorials using the Open in Colab button at the top of each tutorial page. To use Argilla with Google Colab, the recommended method is to use Hugging Face Spaces, as [described in this guide](huggingface-spaces.md). This is the easiest way to set up Argilla for use with Google Colab. For other installations, we recommend you to check the [deployments guide](cloud_providers.md).
+
+If you want to deploy and run Argilla entirely on Google Colab, check [this tutorial by Moritz Laurer(../../../tutorials/notebooks/deploying-textclassification-colab-activelearning.ipynb).
```
Comment (tomaarsen) [inline docs/_source/getting_started/installation/deployments/google-colab.md]:
```suggestion
If you want to deploy and run Argilla entirely on Google Colab, check [this tutorial by Moritz Laurer](../../../tutorials/notebooks/deploying-textclassification-colab-activelearning.ipynb).
```

File: docs/_source/getting_started/installation/deployments/google-colab.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Google Colab
 Google Colab is a free tool that allows users to run and share Jupyter notebooks on Google's servers. It can be useful for quickly trying out models and using GPUs for training and prediction tasks. In the context of Argilla, it can be used for experimenting with pre-trained models, active learning, and computing text embeddings.
 
-To use Argilla with Google Colab, the recommended method is to use Hugging Face Spaces, as [described in this guide](huggingface-spaces.md). This is the easiest way to set up Argilla for use with Google Colab. For other installations, we recommend you to read the [cloud providers guide](cloud_providers.md).
+You can run all Argilla tutorials using the Open in Colab button at the top of each tutorial page. To use Argilla with Google Colab, the recommended method is to use Hugging Face Spaces, as [described in this guide](huggingface-spaces.md). This is the easiest way to set up Argilla for use with Google Colab. For other installations, we recommend you to check the [deployments guide](cloud_providers.md).
```
Comment (tomaarsen) [inline docs/_source/getting_started/installation/deployments/google-colab.md]:
```suggestion
You can run all Argilla tutorials using the Open in Colab button at the top of each tutorial page. To use Argilla with Google Colab, the recommended method is to use Hugging Face Spaces, as [described in this guide](huggingface-spaces.md). This is the easiest way to set up Argilla for use with Google Colab. For other installations, we recommend the [deployments guide](cloud_providers.md).
```
This one is personal preference. These also work:
* we recommend for you to check the ...
* we recommend that you check the...
* we recommend to check the...

File: docs/_source/getting_started/installation/deployments/google-colab.md (hunk 1 above)
Comment (dvsrepo) [reply docs/_source/getting_started/installation/deployments/google-colab.md]: 💯  good catch @tomaarsen:

it should be:

If you want to deploy and run Argilla entirely on Google Colab, check [this tutorial by Moritz Laurer](../../../tutorials/notebooks/deploying-textclassification-colab-activelearning.ipynb).

Right?

Comment (tomaarsen) [reply docs/_source/getting_started/installation/deployments/google-colab.md]: Yes, whoops! I edited it a minute ago :)

Comment (dvsrepo) [reply docs/_source/getting_started/installation/deployments/google-colab.md]: 👍

**Comment by dvsrepo** [comment]
@tomaarsen we can see the docs preview here: https://docs.argilla.io/en/docs-update_colab_guide/getting_started/installation/deployments/google-colab.html (it's still building the latest changes) . We have  a channel on Slack where we're notified, but I've seen others posting a comment from a bot to notify the URL when preview docs are built

**Comment by tomaarsen** [comment]
I know :) I'm looking at the docs preview now, I found the URL via the Slack already. In truth, that is how I found out that the `]` was missing 😄

**Review by tomaarsen — APPROVED** [review]

**Pull request merged**